metadata-db-app/src/cmr/metadata_db/data/oracle/granule_table.clj

```diff
@@ -107,32 +106,24 @@
   (jdbc/db-do-commands db (format "CREATE INDEX %s_crtid ON %s (concept_id, revision_id, transaction_id)"
                                   table-name
                                   table-name))
-
   ;; This index is needed when bulk indexing granules within a collection
   (jdbc/db-do-commands db (format "CREATE INDEX %s_cpk ON %s (parent_collection_id, id)"
                                   table-name
                                   table-name))
-
   ;; This index makes it faster to find counts by collection of undeleted granules for metadata db holdings
-  (jdbc/db-do-commands
-    db (format "create index %s_pdcr on %s (parent_collection_id, deleted, concept_id, revision_id)"
-               table-name
-               table-name)))
+  (jdbc/db-do-commands db (format "create index %s_pdcr on %s (parent_collection_id, deleted, concept_id, revision_id)"
+                                  table-name
+                                  table-name))
+  ;; This index makes it much faster to find granules that are not deleted that have a delete time.
+  (jdbc/db-do-commands db (format "create index %s_dd on %s (deleted, delete_time)"
```

Comment:
Don't you need a different index for small_providers, one that includes the provider_id?

Reply:
Yes, Thank you for catching this!  I have fixed the mistake.